README.md

```diff
@@ -54,10 +54,16 @@ kubectl create -f https://raw.githubusercontent.com/minio/minio-operator/master/
 After you have a distributed MinIO Cluster running (zones.server > 3), you can expand the MinIO cluster using
 
 ```
-kubectl patch minioinstances.miniocontroller.min.io minio --patch "$(cat examples/patch.yaml)" --type=merge
+kubectl patch minioinstances.miniooperator.min.io minio --patch "$(cat examples/patch.yaml)" --type=merge
 ```
 
-You can further keep adding new zones in the `patch.yaml` file and apply the patch, to add new nodes to existing cluster. 
+You can further keep adding new zones in the `patch.yaml` file and apply the patch, to add new nodes to existing cluster.
+
+**NOTE**: Important point to consider _before_ using cluster expansion:
+
+Ensure every Persistent Volume bound to MinIO Operator PVC's has reclaim policy as `retain`. If you attempt cluster expansion while the Persistent Volume reclaim policy is set to something else, it may lead to data loss. If you have the reclaim policy set to something else, change it as explained in [Kubernetes documents](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/).
```

Comment:
Can this be elaborated a bit better from the point of view that PV creation is an important step?

Reply:
Reworded the para, PTAL